R/make-age-polys.R

```diff
@@ -53,7 +53,7 @@ dh.makeAgePolys <- function(df = NULL, age_var = NULL,
   }
 
   if (checks == TRUE) {
-    .isDefined(df = df, vars = vars, conns = conns)
+    .isDefined(df = df, vars = age_vars, conns = conns)
```

Comment:
```suggestion
    .isDefined(df = df, vars = age_var, conns = conns)
```

`age_vars` seems not to be defined

Reply:
Thanks - DONE